packages/support/lib/util.js

```diff
@@ -288,16 +291,14 @@ async function isSameDestination (path1, path2, ...pathN) {
 /**
  * Coerces the given number/string to a valid version string
  *
- * @param {string|number} ver - Version string to coerce
- * @param {boolean} strict [true] - If true then an exception will be thrown
- * if `ver` cannot be coerced
+ * @param {string} ver - Version string to coerce
  * @returns {string} Coerced version number or null if the string cannot be
  * coerced and strict mode is disabled
  * @throws {Error} if strict mode is enabled and `ver` cannot be coerced
  */
-function coerceVersion (ver, strict = true) {
+function coerceVersion (ver) {
```

Comment:
this is a breaking change, which might render other depending code to fail. Are you sure you want to keep it?

Reply:
no, there is no other code calling `coerceVersion` afaik.

Reply:
There is. A simple search through the appium org looks like

<img width="791" alt="coerce" src="https://user-images.githubusercontent.com/7767781/154356383-0b153bf8-961c-4931-bb9a-7d663e8c1cd6.png">

Reply:
oh... ugh.

Reply:
I think I thought this was a different, non-public utils...

Reply:
Anyway, I've reverted this and fixed the type.